### PR TITLE
Made New Readme

### PR DESCRIPTION
**Pull request by avincodes**
@BuddhikaD PLease take a quick look at this! #24 
The new readme is done, hope it is good, and do lemme know if there are any issues!

Thank You :octocat: 
Avin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,46 @@
-# Getting Started with Create React App
+# NSBM FOSS Volunteer
+
+<h1> Hey There <img src = "https://raw.githubusercontent.com/MartinHeinz/MartinHeinz/master/wave.gif" width = 50px> </h1>
+<img width="75%" align="right" alt="Github" src="https://user-images.githubusercontent.com/66766682/139211917-ac01c431-f463-4bdd-aef9-d9dd5f1f297d.png" />
+
+This is the repository for the <br>
+official FOSS-NSBM Website.<br>
```
Comment (BuddhikaD) [inline README.md]: This is not an official website, need to change. This is a site created to get volunteers to the club.

**Review by BuddhikaD — COMMENTED** [review]
Please change the mentioned one and other LGTM

File: README.md (hunk 1 above)
Comment (avincodes) [reply README.md]: on it!

**Comment by avincodes** [comment]
@BuddhikaD Done thanks! :smile:

**Comment by BuddhikaD** [comment]
Thank you 👏❤

**Comment by avincodes** [comment]
> Thank you 👏❤
@BuddhikaD 
No worries and could you please let me know if there is any else I can do here? 😄

**Pull request merged**